crates/derive/src/combine_api.rs

```diff
@@ -115,11 +117,18 @@ impl cgo::Service for API {
                     s => Some(doc::Pointer::from(s)),
                 };
 
+                let validator = new_validator(&schema_json)?;
+                if validator.schemas().len() != 1 as usize {
```

Comment:
supernit; this check seems a bit superfluous given the context (and Rust will bounds-check already).

Reply:
Good point, will take this out.